tuxedo-core/src/executive.rs

```diff
@@ -239,6 +239,10 @@ impl<
         // Store the transient partial header for updating at the end of the block.
         // This will be removed from storage before the end of the block.
         sp_io::storage::set(HEADER_KEY, &header.encode());
+
+        // Clear the extrinsics list from the previous block.
+        // TODO Remove: currently this is needed because we still need to find a way to clear it in block 0.
+        sp_io::storage::clear(EXTRINSIC_KEY);
```

Comment:
I wasn't expecting to need this. Extrinsics only get noted under that key when they are executed. But the genesis extrinsics (and genesis block in general) are not executed.

My understanding is that when a node starts authoring block 1 they will start with a storage that only contains whatever the genesis builder put there and nothing else.

Reply:
Nvm, I didn't understand the storage use yet.

Reply:
It it used by `assimilate_storage` to send the transactions to the `GenesisBlockBuilder` https://github.com/Off-Narrative-Labs/Tuxedo/blob/fba140db0d4c43fc5441dc6b7167a7807f622e9a/tuxedo-template-runtime/src/lib.rs#L189 
But `GenesisBlockBuilder` can't mutate the storage